### PR TITLE
Simple todo list app

### PR DESCRIPTION
**Pull request by kblueberry**

### CODE REVIEW

File: Khrystyna_Derenivska/src/app/tasks-to-do-list/tasks-to-do-list.component.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-tasks-to-do-list',
+  templateUrl: './tasks-to-do-list.component.html',
+  styleUrls: ['./tasks-to-do-list.component.css']
+})
+export class TasksToDoListComponent {
+  username: string = '';
```
Comment (mefissto) [inline Khrystyna_Derenivska/src/app/tasks-to-do-list/tasks-to-do-list.component.ts:9]: Where do you use the property?

File: Khrystyna_Derenivska/src/app/tasks-to-do-list/tasks-to-do-list.component.html [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<div class="tasks__container">
+    <h3 class="container-name">Tasks to do</h3>
+    <div class="tasks__list">
+        <app-task-item></app-task-item>
```
Comment (mefissto) [inline Khrystyna_Derenivska/src/app/tasks-to-do-list/tasks-to-do-list.component.html:4]: Make a list of tasks with ngFor directive

File: Khrystyna_Derenivska/src/app/new-item/new-item.component.ts [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  }
+
+  ngOnInit(): void {}
+
+  onCreateButtonClick() {
+    this.itemCreated = true;
+    this.itemCreationStatus = `Item was created. Name is ' + this.serverName'`;
+  }
+
+  onUpdateItemName(event: any) {
```
Comment (mefissto) [inline Khrystyna_Derenivska/src/app/new-item/new-item.component.ts:27]: Avoid the "any" type

File: Khrystyna_Derenivska/src/app/new-item/new-item.component.ts [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    setTimeout(() => {
+      this.allowNewItem = true;
+    }, 2000);
+  }
+
+  ngOnInit(): void {}
+
+  onCreateButtonClick() {
+    this.itemCreated = true;
+    this.itemCreationStatus = `Item was created. Name is ' + this.serverName'`;
```
Comment (mefissto) [inline Khrystyna_Derenivska/src/app/new-item/new-item.component.ts:24]: If you use a template string you can write something like that --> `Item was created. Name is ${this.serverName}`

File: Khrystyna_Derenivska/src/app/new-item/new-item.component.ts [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+  styleUrls: ['./new-item.component.css']
+})
+export class NewItemComponent implements OnInit {
+  allowNewItem = false;
+  itemCreationStatus = 'No item was created';
+  serverName = 'Test task';
+  itemCreated = false;
+
+  constructor() {
+    setTimeout(() => {
```
Comment (mefissto) [inline Khrystyna_Derenivska/src/app/new-item/new-item.component.ts:15]: Clear timeout

**Comment by kblueberry** [comment]
Going to open PR from repo, not from fork. Closing this PR

File: Khrystyna_Derenivska/src/app/task-item/task-item.component.html [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+            <label for="list__item-1_value">{{task.name}}</label>
+        </div>
+    </ng-template>
+
+
+    <div class="list__item_description">
+        <div *ngIf="isEditMode; then saveButton else editButton"></div>
+        <ng-template #editButton>
+            <button class="button button__edit" (click)="onEditClick()"
+                [attr.disabled]="task.isDone ? '' : null">Edit</button>
```
Comment (mefissto) [inline Khrystyna_Derenivska/src/app/task-item/task-item.component.html:18]: Why do you use '' and null as value? If you need to set a disable attribute you can use only property "task.isDone" without a ternary operator.

**Pull request closed without merging**